sdk/documentintelligence/Azure.AI.DocumentIntelligence/CHANGELOG.md

```diff
@@ -3,12 +3,50 @@
 ## 1.0.0-beta.3 (Unreleased)
 
 ### Features Added
+- Added support for the Analyze Batch Documents API:
```

Comment:
This is generally not the pattern that we prefer for change logs, but is consistent with history for this library, so I'll just mention and move on.  😄

Reply:
For reference, what's the pattern we usually prefer?

Reply:
Limiting to high-level features.   Something like:

- Added the ability to analyze Starbucks receipts.
- Fixed an error that caused a timeout when analyzing newspapers.

Reply:
Detailing every single public API member adds a lot of noise, making it harder for developers to find things they care about.

Reply:
My understanding was that the CHANGELOG was supposed to be an exhaustive list of changes, while we have highlighted release notes published separately.

I'll keep the CHANGELOG the way it is for this release due to time constraints, but we can consider following the preferred approach on the next one.